Block/TurnToConfig.php

```diff
@@ -85,16 +85,13 @@ public function getJavaScriptConfig()
         $additionalConfigData['extensionVersion'] = ['magentoVersion'=> $this->versionHelper->getMagentoVersion(), 'turnToCart' => $this->versionHelper->getTurnToVersion()];
         $additionalConfigData['baseUrl'] = $this->_storeManager->getStore()->getBaseUrl();
         $additionalConfigData['siteKey' ] = $this->configHelper->getSiteKey();
+        $additionalConfigData['sso'] = ['userDataFn' => null];
 
         if ($this->configHelper->getQaEnabled()) {
             $additionalConfigData['qa'] = [];
         }
 
-        if($this->configHelper->getSsoEnabled()){
-            $additionalConfigData['sso'] = ['userDataFn' => null];
-        }
-
-        if ($this->configHelper->getCommentsPinboardTeaserEnabled() ) {
+        if ($this->configHelper->getCheckoutCommentsEnabled() ) {
```

Comment:
Hey @joshfedo, can you change this line back? I specifically added a setting to enabled the Checkout Comments Pinboard Teaser